ETC.tsv

```diff
@@ -11227,7 +11227,7 @@ ETC_20150428_011228	Panto Archer
 ETC_20150428_011229	Yellow Zigri
 ETC_20150428_011230	Dionis
 ETC_20150428_011231	Disgusting Deathweaver
-ETC_20150428_011232	Pyrokinetic Kelberos
+ETC_20150428_011232	Pyrokinetic Kerberos
```

Comment:
Kerberos is probably Cerberus. I can confirm this personally by seeing the boss once and actually being a large (multi-headed) hound